src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -246,17 +247,18 @@ def _get_posix_test_path_string(self, path=None):
         """
         return " ".join(self._get_posix_test_path_list(path_list=path))
 
-    def new_posix_test_path(self, shared=False, create=True, parent=None, mount_dir=False):
+    def new_posix_test_path(self, shared=False, create=True, parent=None, mount_dir_size=False):
```

Comment:
Why is mount_dir_size set to false ? It is a string... Could be None...

Reply:
Oops! It used to be a `bool` but now it's a `str`. I forgot to change this. Thanks :)